app/components/new-reminder.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  store: Ember.inject.service(),
```

Comment:
We had problems getting the new reminder to render when we put it in the controller.  I'd like to figure out how to do both.